daemon/cluster/convert/service.go

```diff
@@ -170,6 +176,11 @@ func ServiceSpecToGRPC(s types.ServiceSpec) (swarmapi.ServiceSpec, error) {
 		}
 		spec.Task.Runtime = &swarmapi.TaskSpec_Container{Container: containerSpec}
 	case types.RuntimePlugin:
+		if s.Mode.Replicated != nil {
+			return swarmapi.ServiceSpec{}, errors.New("plugins must not use replicated mode")
```

Comment:
I'm curious if there's anything inherently wrong with a replicated plugin, or if that's just not how we expect the feature to be used.